src/core/surface/channel.c

```diff
@@ -235,6 +257,18 @@ grpc_mdstr *grpc_channel_get_status_string(grpc_channel *channel) {
   return channel->grpc_status_string;
 }
 
+grpc_mdelem *grpc_channel_get_reffed_status_elem(grpc_channel *channel, int i) {
+  if (i >= 0 && i < NUM_CACHED_STATUS_ELEMS) {
```

Comment:
nit: given i >= 0 is more likely to be true than i < NUM..., flipping the comparisons could save the i >= 0 one if i >= NUM...